v2/yarpctransport/handler_test.go

```diff
@@ -25,36 +25,37 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	yarpc "go.uber.org/yarpc/v2"
 	"go.uber.org/zap/zapcore"
 )
 
-type UnaryHandlerFunc func(context.Context, *Request) (*Response, error)
-type StreamHandlerFunc func(*ServerStream) error
+type UnaryHandlerFunc func(context.Context, *yarpc.Request, *yarpc.Buffer) (*yarpc.Response, *yarpc.Buffer, error)
+type StreamHandlerFunc func(*yarpc.ServerStream) error
 
-func (f UnaryHandlerFunc) Handle(ctx context.Context, r *Request) (*Response, error) {
-	return f(ctx, r, w)
+func (f UnaryHandlerFunc) Handle(ctx context.Context, r *yarpc.Request, b *yarpc.Buffer) (*yarpc.Response, *yarpc.Buffer, error) {
+	return f(ctx, r, b)
```

Comment:
Curious why there isn't syntax highlighting here. I wonder if there is a compilation issue somewhere?

Reply:
another test package is importing `yarpctest` which still has some work needed, I think that's what's affecting this file.